.circleci/integration-tests/master_dag.py

```diff
@@ -163,6 +177,19 @@ def prepare_dag_dependency(task_info, execution_time):
     get_airflow_version = BashOperator(
         task_id="get_airflow_version", bash_command="airflow version", do_xcom_push=True
     )
+    #
+    check_logs_data = PythonOperator(
+        task_id="check_logs",
+        python_callable=log_checker,
+        op_args=[
+            "get_airflow_version",
+            "{{ ti.xcom_pull(task_ids='get_airflow_version') }}",
+            "this_string_should_not_be_present_in_logs",
+        ],
+    )
+
+    airflow_version_check = (get_airflow_version, check_logs_data)
```

Comment:
May I know why is `check_logs_data` a part of `airflow_version_check` and why do we need `airflow_version_check` as a tuple?

Reply:
@Lee-W  i have added check_log_data as part of airflow_version because in check logs we are validating logs of airflow version task only, also to ensure the check_logs is downstream of airflow_version_task I have made it a tuple so that I can set [upstream](https://github.com/astronomer/astronomer-providers/pull/1246/files#diff-14b9f739868be8ddebc40caaf0868cbc33ec9a63fdfebceb3eb723b413d396eaR357) and [downstream](https://github.com/astronomer/astronomer-providers/pull/1246/files#diff-14b9f739868be8ddebc40caaf0868cbc33ec9a63fdfebceb3eb723b413d396eaR357)

Reply:
got it 👍